pennylane_lightning/core/src/simulators/lightning_kokkos/measurements/tests/Test_StateVectorKokkos_Expval.cpp

```diff
@@ -381,6 +381,60 @@ TEMPLATE_TEST_CASE("Test expectation value of TensorProdObs",
     }
 }
 
+TEMPLATE_TEST_CASE("Test expectation value of TensorProdObs shots",
```

Comment:
And then we may be able to move these tests to the base class.

Reply:
Done!